test/integration/targets/incidental_setup_mysql_db/vars/Suse.yml

```diff
@@ -4,3 +4,9 @@ mysql_packages:
     - mariadb
     - python-PyMySQL
     - bzip2
+
+# opensuse15py2 brings in a bunch of py3 stuff. Nuke it.
+mysql_cleanup_packages:
+    - python3-base
+    - python3-mysqlclient
+    - libpython3_6m1_0
```

Comment:
There may be collections which will need this same change to their tests.

Reply:
fwiw, I'm in the process of removing this incidental.